dhall-yaml/src/Dhall/Yaml.hs

```diff
@@ -51,26 +51,23 @@ jsonToYaml
     -> Bool
     -> ByteString
 jsonToYaml json documents quoted =
+  let encoder = Data.YAML.Aeson.encodeValue' schemaEncoder YT.UTF8 in
```

Comment:
(Minor style issue): Since there's already a  `where` clause, I would find it hard to decide whether to add the next definition there or as a let-binding. I'd suggest defining `encoder` in the `where`-clause therefore.